### PR TITLE
New package: drive-0.3.8.1

### PR DESCRIPTION
**Pull request by arete**
Signed-off-by: Gerardo Di Iorio <arete74@gmail.com>

### CODE REVIEW

File: srcpkgs/drive/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'drive'
+pkgname=drive
+version=0.3.7
+revision=1
+build_style=go
+hostmakedepends="git go"
```
Comment (ebfe) [inline srcpkgs/drive/template]: Can remove go, that's implicit by the build_style.

File: srcpkgs/drive/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+short_desc="Google Drive client for the commandline"
+maintainer="Gerardo Di Iorio <arete74@gmail.com>"
+license="Apache-2.0"
+homepage="https://github.com/odeke-em/drive"
+go_import_path="github.com/odeke-em/drive"
+go_package="github.com/odeke-em/drive/cmd/drive github.com/odeke-em/drive/drive-server "
+distfiles="$homepage/archive/v$version.tar.gz"
+checksum=f14b3279bb2859fcb565f4fa6c651e412b2b8007f9079d16c68791e3ff27735a
```
Comment (ebfe) [inline srcpkgs/drive/template]:
```
post_install() {
	vlicense LICENSE
}
```

Comment (ebfe) [inline srcpkgs/drive/template]: Ah stop Apache-2.0 so probably not needed.

**Comment by ebfe** [comment]
Maybe `vdoc README.md`.

**Comment by arete** [comment]
Hi, i have update pull request with you suggest.
regards

**Pull request merged**